int32 frameID

geometry_msgs/Pose pose

float64 markerPixelWidth

float64[] mags
float64[] phases

float64 radius
float64 z
float64 oop_rotation

bool hasSignature
bool hasValidXORs
bool hasValidCRC

string payloadOct
string xorBin

uint64 signature
uint64 CRC12Expected
uint64 CRC12Decoded